k8s/deploy_app.sh

```diff
@@ -9,6 +9,12 @@ fi
 
 DOCKER_REGISTRY="${DOCKER_REGISTRY:-eu.gcr.io/census-eq-ci}"
 IMAGE_TAG="${IMAGE_TAG:-latest}"
+REQUESTED_CPU="${REQUESTED_CPU:-3}"
+MAX_UNAVAILABLE="${MAX_UNAVAILABLE:-1}"
+MAX_SURGE="${MAX_SURGE:-1}"
```

Comment:
It's not clear these are for rolling updates, it would be better to prefix the two with `ROLLING_UPDATE_`.